_data/education2.yml

```diff
@@ -0,0 +1,9 @@
+- course: Software Developer 
+  institute: Microverse
+  dates: 2020 - 2021
+  logo: uob.png
+
+- course: B.Eng Agricultural Engineering, Major in Farm Mechanization
+  institute: University of Uyo
+  dates: 2005 - 2010
+  logo: uob.png
```

Comment:
Let's remove this one, and allow only the Software Developer only

Reply:
Okay!